### PR TITLE
Change Windows UI Toolkit to Windows UI Library

### PR DESCRIPTION
**Pull request by thomasclaudiushuber**
It's called Windows UI Library. Windows UI Toolkit could be confusing for developers, as there's a Windows Community Toolkit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by stmoy — APPROVED** [review]
FWIW, it looks like both the old and new links both go to the same location - double checking that this is expected.

We could also consider directing folks to https://aka.ms/winui 

CC: @anawishnoff - do you have a preference?

**Comment by anawishnoff** [comment]
@stmoy @thomasclaudiushuber 
> FWIW, it looks like both the old and new links both go to the same location - double checking that this is expected.
> 
> We could also consider directing folks to https://aka.ms/winui
> 
> CC: @anawishnoff - do you have a preference?

Thanks for making this change! I'd recommend linking to our [overview docs page](https://docs.microsoft.com/windows/apps/winui/) as it goes over both WinUI 2 and 3. That way, this change could work for a WinUI 3 XCG as well.

File: README.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ description: Demonstrates the usage of all XAML/WinUI controls in an interactive
 
 [![Build Status](https://dev.azure.com/stmoy/Xaml%20Controls%20Gallery/_apis/build/status/XAML%20Controls%20%20Gallery%20-%20CI%20Tests?branchName=master)](https://dev.azure.com/stmoy/Xaml%20Controls%20Gallery/_build/latest?definitionId=2&branchName=master)
 
-Shows all of the XAML controls in an interactive format. This app is the interactive companion to the [Fluent Design Guidelines](https://docs.microsoft.com/windows/uwp/design/basics/) and shows the usage of both UWP Xaml APIs and [Windows UI Toolkit](https://docs.microsoft.com/uwp/toolkits/winui/) APIs.
+Shows all of the XAML controls in an interactive format. This app is the interactive companion to the [Fluent Design Guidelines](https://docs.microsoft.com/windows/uwp/design/basics/) and shows the usage of both UWP Xaml APIs and [Windows UI Library](https://docs.microsoft.com/windows/apps/winui/winui2/) APIs.
```
Comment (stmoy) [inline README.md:20]:
```suggestion
Shows all of the XAML controls in an interactive format. This app is the interactive companion to the [Fluent Design Guidelines](https://docs.microsoft.com/windows/uwp/design/basics/) and shows the usage of both UWP Xaml APIs and [Windows UI Library](https://docs.microsoft.com/windows/apps/winui/) APIs.

```

File: README.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ description: Demonstrates the usage of all XAML/WinUI controls in an interactive
...
 
 ## The XAML Controls Gallery shows how to:
 
 - **Specify XAML controls in markup:** Each control page shows the markup used to create each example.
-- **Use the Microsoft.UI.Xaml (WinUI) Library:** The app includes the latest WinUI NuGet package and shows how to use the [Windows UI Toolkit](https://docs.microsoft.com/uwp/toolkits/winui/) controls like NavigationView, SwipeControl, and more.
+- **Use the Microsoft.UI.Xaml (WinUI) Library:** The app includes the latest WinUI NuGet package and shows how to use the [Windows UI Library](https://docs.microsoft.com/windows/apps/winui/winui2/) controls like NavigationView, SwipeControl, and more.
```
Comment (stmoy) [inline README.md:25]:
```suggestion
- **Use the Microsoft.UI.Xaml (WinUI) Library:** The app includes the latest WinUI NuGet package and shows how to use the [Windows UI Library](https://docs.microsoft.com/windows/apps/winui/) controls like NavigationView, SwipeControl, and more.

```

**Comment by thomasclaudiushuber** [comment]
Yep, good point @anawishnoff. Originally I thought about linking to WinUI Overview, but the fact that XCG still uses WinUI 2.x made me link to WinUI 2.x. But I guess it's better to link to the overview. Thank you, and thanks @stmoy for the commit suggestions.

**Review by stmoy — APPROVED** [review]
Thank you for helping to clarify this @thomasclaudiushuber!

**Pull request merged**